packages/api/core/src/api/import.ts

```diff
@@ -162,7 +162,13 @@ export default async ({
                 delete packageJSON.devDependencies[key];
               }
             }
-          }
+          };
+
+          await Promise.all(
+            keys.map(async (key) => {
+              await removeBuildToolDependencies(key);
+            })
+          );
```

Comment:
```suggestion
          await Promise.all(
            keys.map(removeBuildToolDependencies)
          );
```

To facilitate maintenance, we should `return`.

```ts
          const result = await Promise.all(
            keys.map(async (key) => {
              await removeBuildToolDependencies(key);
            })
          );

          console.log(result); // => [undefined, undefined, ...]
```

Reply:
This is incorrect and can result in invalid behavior `map` passed both `key` and `index` as arguments. If the function takes a second argument this is unsafe behavior.  Having the smaller `(key) => doThing(key)` wrapper is better and safer

Reply:
Just wanted to make sure I'm clear- which is the intended change request?

Reply:
```ts
await Promise.all(
  keys.map((key) => removeBuildToolDependencies)
);
```

We should leave the parallelize to `Promise.all`, not `map`. 
`map` is responsible for returning a `promise object`. `promise.all` is responsible for parallelize.

Storybook code:
<img width="641" alt="image" src="https://user-images.githubusercontent.com/8198408/201251587-f751605d-3aa7-43d9-b6a3-2baa82c0a2c5.png">

Reply:
At first the code I suggested was:

```ts
await Promise.all(
  keys.map(removeBuildToolDependencies)
);
```

@MarshallOfSound found some problems with this code, so I changed it to:

```ts
await Promise.all(
  keys.map((key) => removeBuildToolDependencies(key))
);
```